docs/guides/miscellaneous/migration_guide.md

```diff
@@ -185,13 +185,9 @@ Migrating Datasource configurations that contain connections to the cloud or dat
 
 ### Manually Migrate V2 Checkpoints to V3 Checkpoints
 
-In Great Expectations version 0.13.7, we introduced an improved Checkpoints feature, which allowed Checkpoints to utilize features the V3 API. As a result, Checkpoints are now able to [filter and sort batches from configured datasources](/docs/guides/connecting_to_your_data/how_to_get_a_batch_of_data_from_a_configured_datasource), [introspect and partition tables as batches](/docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql), with multi-batch Expectations soon to come.  As part of these design improvements, Validation Operators (originally located in the `great_expectations.yml` file) were combined into the Checkpoint configuration itself as `action_list` items.
+In Great Expectations version 0.13.7, we introduced an improved Checkpoints feature, which allowed Checkpoints to utilize features the V3 API. As a result, Checkpoints are now able to [filter and sort batches from configured datasources](/docs/guides/connecting_to_your_data/how_to_get_a_batch_of_data_from_a_configured_datasource), [introspect and partition tables as batches](/docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql), with multi-batch Expectations soon to come.  As part of these design improvements, Validation Operators (originally located in the `great_expectations.yml` file) were combined into Checkpoint configurations. 
 
-:::note Note on V3 Checkpoints and Validation Operators
-
-  In V2, you were able to run Validation Operators directly from the DataContext, but in V3 the functionality has been migrated to the action_list field in Checkpoints. This change offers a convenient abstraction for running Validations and ensures that all actions associated with running validations are included in one place, rather than split up between the great_expectations.yml file and Checkpoint configuration. 
-
-:::
+This means that, although Validation Operators were run directly from the DataContext in V2, they are now run by Checkpoints in V3 as part of `action_list` items. This change offers a convenient abstraction for running Validations and ensures that all actions associated with running validations are included in one place, rather than split up between the `great_expectations.yml` file and Checkpoint configuration.
 
```

Comment:
@spencerhardwick since I am adding another `Note` section below, I tried combining this `Note` with the text above so that we don't have multiple notes for a single section. Could you let me know what you think? I'd be happy to revert back to what you had before if yo'ud like